virttest/libvirt_cgroup.py

```diff
@@ -154,12 +155,15 @@ def __get_cg_file_path(cg_key, cg_path, cg_file_name):
             dev_init_dict = {"rbps": "max", "wbps": "max", "riops": "max",
                              "wiops": "max"}
             dev_list = []
+            if not self.is_bfq():
+                CGROUP_V1_BLKIO_FILE_MAPPING['weight'] = 'blkio.weight'
+                CGROUP_V1_BLKIO_FILE_MAPPING['weight_device'] = 'blkio.weight_device'
```

Comment:
great, it's better to test weigt_device only in cfq env. this is a simple fix and won't effect cgroup1/2 bfq env.